trusted.ci.jenkins.io.tf

```diff
@@ -503,3 +553,66 @@ resource "azurerm_network_security_rule" "allow_inbound_ssh_from_bounce_to_ephem
   resource_group_name         = data.azurerm_resource_group.trusted.name
   network_security_group_name = azurerm_network_security_group.trusted_ci_controller.name
 }
+
+resource "azurerm_network_security_rule" "allow_inbound_ssh_from_admins_to_bounce" {
+  for_each = local.admin_allowed_ips
+
+  name                        = "allow-inbound-ssh-from-admin-${each.key}-to-bounce"
+  priority                    = 4000 + index(keys(local.admin_allowed_ips), each.key)
+  direction                   = "Inbound"
+  access                      = "Allow"
+  protocol                    = "Tcp"
+  source_port_range           = "*"
+  destination_port_range      = "22"
+  source_address_prefix       = each.value
+  destination_address_prefix  = azurerm_linux_virtual_machine.trusted_bounce.private_ip_address
+  resource_group_name         = data.azurerm_resource_group.trusted.name
+  network_security_group_name = azurerm_network_security_group.trusted_ci_controller.name
+}
+
+resource "azurerm_network_security_rule" "allow_inbound_jenkins_usage_from_vnet_to_controller" {
+  name                  = "allow-inbound-jenkins-usage-from-vnet-to-controller"
+  priority              = 4094
+  direction             = "Inbound"
+  access                = "Allow"
+  protocol              = "Tcp"
+  source_port_range     = "*"
+  source_address_prefix = "VirtualNetwork"
+  destination_port_ranges = [
+    "443",   # Only HTTPS
+    "50000", # Direct TCP Inbound protocol
+  ]
+  destination_address_prefix  = azurerm_linux_virtual_machine.trusted_ci_controller.private_ip_address
+  resource_group_name         = data.azurerm_resource_group.trusted.name
+  network_security_group_name = azurerm_network_security_group.trusted_ci_controller.name
+}
+
+# This rule overrides an Azure-Default rule. its priority must be < 65000.
+resource "azurerm_network_security_rule" "deny_all_inbound_from_internet" {
+  name                        = "deny-all-inbound-from-internet"
+  priority                    = 4095
+  direction                   = "Inbound"
+  access                      = "Deny"
+  protocol                    = "*"
+  source_port_range           = "*"
+  destination_port_range      = "*"
+  source_address_prefix       = "Internet"
+  destination_address_prefix  = "*"
+  resource_group_name         = data.azurerm_resource_group.trusted.name
+  network_security_group_name = azurerm_network_security_group.trusted_ci_controller.name
+}
+
+# This rule overrides an Azure-Default rule. its priority must be < 65000
+resource "azurerm_network_security_rule" "deny_all_inbound_from_vnet" {
+  name                        = "deny-all-inbound-from-vnet"
+  priority                    = 4096 # Maximum value allowed by the
```

Comment:
"by the" what?

Reply:
Erf, automerged...

Reply:
I'll fix it on a subsequent PR, no worries

Reply:
Fixed in https://github.com/jenkins-infra/azure/pull/357